examples/demos/example21-grouping.js

```diff
@@ -1,42 +1,46 @@
+import faker from 'faker';
```

Comment:
I tried to convert it to tsx but it will eventually touch tons of files like grouping related js files, so maybe we should do the conversion later in another PR.